examples/optimized-paper-config/README.md

```diff
@@ -0,0 +1,59 @@
+
+# Repository Proposal for Optimized Minecraft Configurations for Paper
+
+This repository provides a set of optimized configuration files for Minecraft servers, focusing on improving performance and reducing lag. These configurations are based on community best practices and have been tested. You can also use your config file; you just need to replace the env var with another repo. It's important to follow the structure of a config file version like this [repo](https://github.com/Alpha018/paper-config-optimized).
```

Comment:
In the context of the examples directory, this paragraph is confusing since it refers to "this repo" which is actually the repo you're maintaining. 

Perhaps rather than copy over the README.md as is you could include a comment in the compose file that points users to your repo.